css/rjsf.css

```diff
@@ -1,25 +1,25 @@
-.generic-form * {
+.rjsf * {
```

Comment:
Nope, it's acronym for *r*eact *j*son *s*chema *f*orm and is the name of the provided CSS stylesheet root identifier for the built forms.